askui_example/logging/logger.ts

```diff
@@ -1,10 +1,28 @@
 import "@askui/jest-allure-circus";
+import * as os from 'os';
 
 declare const allure: {
     step(name: string, body?: () => void): void;
     attachment(name: string, content: string, type: string): void;
+    writeEnvironmentInfo(info: Record<string, string>): void;
 }
 
+// Generate environment info
+const deviceId = `${os.hostname()}-${Math.random().toString(36).substring(2, 8).toUpperCase()}`;
```

Comment:
I would only use the host name here. The device it is only for android.

```suggestion
const hostname = `${os.hostname()}`;
```

Reply:
Then you have to change line 17  too.